kw/platform/monkey.py

```diff
@@ -0,0 +1,50 @@
+"""
+Monkey
+======
+"""
+
+import importlib
+import threading
+
+
+# Define modules and which of them should be patched automatically.
+_PATCH_MODULES = {
+    "pytest": False,  # !!! serves only for unittests of this package
+    "requests": False,
+}
+_PATCHED_MODULES = set()
+
+
+def _patch_module(module):
+    with threading.Lock():
+        if module in _PATCHED_MODULES:
+            return False
+        path = f"kw.platform.{module}.patch"
```

Comment:
You cannot use the `f` strings as we want to support python 2.7 and 3.5. You need to use `format` method on strings instead. That's why tests are failing.